data/mods/Globally Unique Specials/overmap/overmap_special/specials.json

```diff
@@ -0,0 +1,52 @@
+[
+  {
+    "type": "overmap_special",
+    "id": "Isherwood Farms",
+    "flags": [ "FARM", "GLOBALLY_UNIQUE", "SAFE_AT_WORLDGEN" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "evac_center",
+    "flags": [ "GLOBALLY_UNIQUE", "SAFE_AT_WORLDGEN", "MAN_MADE" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "hub_01",
+    "flags": [ "GLOBALLY_UNIQUE", "SAFE_AT_WORLDGEN" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "isolated_road",
+    "flags": [ "GLOBALLY_UNIQUE", "SAFE_AT_WORLDGEN" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "ranch_camp",
+    "flags": [ "GLOBALLY_UNIQUE", "SAFE_AT_WORLDGEN", "MAN_MADE" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "valhalla_cult",
+    "flags": [ "CLASSIC", "GLOBALLY_UNIQUE", "MAN_MADE" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "Occupied Lumbermill",
+    "flags": [ "CLASSIC", "GLOBALLY UNIQUE", "WILDERNESS" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "Occupied Scrap Yard",
+    "flags": [ "CLASSIC", "GLOBALLY UNIQUE" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "Occupied Chem Lab",
+    "flags": [ "CLASSIC", "GLOBALLY UNIQUE" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "Occupied Homeless Camp",
+    "flags": [ "CLASSIC", "GLOBALLY UNIQUE", "URBAN" ]
```

Comment:
```suggestion
    "flags": [ "CLASSIC", "URBAN" ]
```
Here too.

Reply:
Could they at least be UNIQUE, to keep them off of the same OM tile? That's what that flag does, right? The names might be generic but they still have identical railroaded text and missions.

Reply:
Yes...it does. As things stand, can two of the exact same trader appear on the same overmap chunk? Is that not what UNIQUE prevents?

Reply:
So I'll summarize my understanding with this question: do you need to see two occupied lumber mills in the same overmap chunk, doing and saying exactly the same thing and offering the same quests in the same order?

Reply:
UNIQUE prevents two (in this example, occupied lumbermills) from appearing in the same overmap chunk. However, multiple can spawn in the same world. GLOBALLY_UNIQUE only allows for one to spawn per world (you will only ever see it once in a game) If it is only UNIQUE, you can see multiple per game.

Reply:
Well, when nested mapgen has been taked such advantage of as to create that variety and mutability, that will be a fine day indeed! Instead, we are here as we are today. 

What satisfaction are you getting out of multiple identical traders on the same overmap? In a mod, not even suggested as active-by-default, that I'm trying to push?

Reply:
Multiple traders just seem logical to me. However, on reflection, I think I should try the nested mapgen idea for a bit more variety. On further reflection, I think the idea of just making them overmap-unique would work better.

Reply:
Huh. Out of curiosity, how likely is a lumber mill to be occupied as opposed to unoccupied?

Reply:
one out of every two lumber mills is a fortress, I think? Lumberjacks become the most numerous living humans left on Earth?

Reply:
Sorry I didn't get back to you sooner.

Reply:
I leave it to you to either insert occurrences that you think are appropriate either in the changes you're suggesting to this PR or directly to the main repo, if you like.

Promptness isn't an issue lol, no worries. Hopefully the content freeze doesn't descend before I can at least present this to the cabal.

Reply:
Um, also though, it says "outdated suggestions cannot be applied", dunno if there's something I'm supposed to do about that.

Reply:
I wouldn't worry about that. I'll delete the suggested code and make a new one.